backend/mora/service/association.py

```diff
@@ -77,9 +80,10 @@ def prepare_create(self, req):
             funktionstype=association_type_uuid,
             opgaver=[
                 {'uuid': job_function_uuid}] if job_function_uuid else None,
-            adresser=[
-                address.get_relation_for(address_obj),
-            ] if address_obj else None,
+            adresser=[{
+                'uuid': addr_id,
+                'objekttype': 'organisationfunktion'
```

Comment:
This feels vaguely iffy… We seem to be using the `adresser` relation for something rather unrelated, i.e. pointing to an `organisationfunktion`. IIRC we don't do that for IT systems, which went through a similar iteration.

Reply:
I based it on your previous code, but I see your point :smile: 
I'll change it.